public/app/sections/header-navbar/app-preferences/components/app-preferences-users/templates/app-preferences-users.template.html

```diff
@@ -54,6 +54,7 @@ <h3 class="setting__main-title">Users</h3>
             </tr>
           </tfoot>
         </table>
+        <smart-table-with-dynamic-options></smart-table-with-dynamic-options>
```

Comment:
The current line must be removed if the following commit is removed https://github.com/sipcapture/homer-app/pull/62/commits/5d513bff469ba6623e169589e5e81a62168f226a